functions.py

```diff
@@ -11,106 +11,118 @@
 import matplotlib.pyplot as plt
 
 try:
-  from google.colab import files
-  IN_COLAB = True
+    from google.colab import files
```

Comment:
we don't need this check (colab code), can you remove?

Reply:
preferably separate PR to ensure google colab functionality is eradicated entirely from this script